six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -306,12 +308,13 @@ def readRegion(inputPathname, startRow, numRows, startCol, numCols, schemaPaths
     getWidebandRegion(inputPathname, schemaPaths, complexData, startRow, numRows, startCol, numCols, widebandBuffer)
 
     return widebandData, complexData
+
+
 def writeAsNITF(outFile, schemaPaths, complexData, image):
     writeNITF(outFile, schemaPaths, complexData,
         image.__array_interface__["data"][0])
 
 def readFromNITF(pathname, schemaPaths=VectorString()):
-    pathname = pathname + ".nitf"
```

Comment:
There were actually a number of read/write functions that append extensions, so I wouldn't say I put it there by accident. But this is an intentional removal, yes.